MediaTracker/app/src/main/java/com/izor066/android/mediatracker/ui/adapter/ItemAdapter.java

```diff
@@ -11,21 +11,36 @@
 
 import com.izor066.android.mediatracker.MediaTrackerApplication;
 import com.izor066.android.mediatracker.R;
-import com.izor066.android.mediatracker.api.DataSource;
 import com.izor066.android.mediatracker.api.model.Book;
 import com.squareup.picasso.Picasso;
 
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
 /**
  * Created by igor on 13/11/15.
  */
-public class ItemAdapter extends RecyclerView.Adapter<ItemAdapter.ItemAdapterViewHolder>  {
+public class ItemAdapter extends RecyclerView.Adapter<ItemAdapter.ItemAdapterViewHolder> {
+
+    private String currentSortCriteria = "added"; // "title", "author"
```

Comment:
This `currentSortCriteria` can be an `enum` in order to avoid duplication and to keep strong typing. 

```java
public enum SortCriteria {
   ADDED,
   TITLE,
   AUTHOR
}
```